MdeModulePkg/MdeModulePkg.dsc

```diff
@@ -158,15 +158,12 @@
 ## MU_CHANGE - SmmLockBoxPeiLib only supports IA32 and X64.
   LockBoxLib|MdeModulePkg/Library/LockBoxNullLib/LockBoxNullLib.inf
   PcdDatabaseLoaderLib|MdeModulePkg/Library/PcdDatabaseLoaderLib/Pei/PcdDatabaseLoaderLibPei.inf  # MU_CHANGE
+  DevicePathLib|MdePkg/Library/UefiDevicePathLib/UefiDevicePathLibBase.inf  # MU_CHANGE

```

Comment:
I think if you move this right under LockBoxLib we can drop the MU_CHANGE tag

Reply:
Moved, Thanks!